### PR TITLE
Added price parameter to product

### PR DESCRIPTION
**Pull request by Ladyslav**

### CODE REVIEW

File: MyStore/src/GeneratorData.cpp [hunk 1]
```diff
@@ -9,9 +9,11 @@ InfoProduct GeneratorData::GeneratedProduct()
 
 	int index_color = rand() * ((max_index_colors - 0 + 0.999) / RAND_MAX) + 0;
 	int index_name = rand() * ((max_index_names - 0 + 0.999) / RAND_MAX) + 0;
+	srand(time(NULL));
 
 	data.m_product_color = m_colors[index_color];
 	data.m_product_name = m_names[index_name];
+	data.m_price = (rand() + rand()) / 130 * 2;
```
Comment (vnest-ss) [inline MyStore/src/GeneratorData.cpp]: We must be able to control minimum and maximum price of the product. Please add min and max price parameters and retrieve here random price in this range.

Comment (Ladyslav) [reply MyStore/src/GeneratorData.cpp]: Got it. I've already pushed changes.

File: MyStore/src/GeneratorData.cpp [hunk 2]
```diff
@@ -10,8 +10,10 @@ InfoProduct GeneratorData::GeneratedProduct()
 	int index_color = rand() * ((max_index_colors - 0 + 0.999) / RAND_MAX) + 0;
 	int index_name = rand() * ((max_index_names - 0 + 0.999) / RAND_MAX) + 0;
 
+
```
Comment (vnest-ss) [inline MyStore/src/GeneratorData.cpp:13]: Nit: redundant empty line.

File: MyStore/Settings.cpp [hunk 3]
```diff
@@ -0,0 +1 @@
+#include "Settings.h"
```
Comment (vnest-ss) [inline MyStore/Settings.cpp]: Settings.h and Settings.cpp files are irrelevant to this change, let's remove them from the change.

**Review by vnest-ss — COMMENTED** [review]
Few more comments on this change. And please squash two commits into one.

**Review by vnest-ss — APPROVED** [review]
Looks and works good!

**Pull request merged**